spanner/integration_test.go

```diff
@@ -288,22 +290,51 @@ var (
             PRIMARY KEY (CartId))`,
 	}
 
+	bitReverseSeqDBStatments = []string{
+		`CREATE SEQUENCE seqT OPTIONS (sequence_kind = "bit_reversed_positive")`,
+		`CREATE TABLE T (
+            id INT64 DEFAULT (GET_NEXT_SEQUENCE_VALUE('seqT')),
```

Comment:
I think this will not work due to some backend changes. Can you check this Rahul?
```suggestion
            id INT64 DEFAULT (GET_NEXT_SEQUENCE_VALUE(Sequence seqT)),
```

Reply:
This should have thrown error like 
`Query failed: No matching signature for function SPANNER:GET_INTERNAL_SEQUENCE_STATE for argument types: STRING `